grafana-plugin/src/containers/UsersTimezones/UsersTimezones.tsx

```diff
@@ -277,4 +305,45 @@ const AvatarGroup = (props: AvatarGroupProps) => {
   );
 };
 
+function getColorSchemeMappingForUsers(store: RootStore, scheduleId: string): { [userId: string]: Set<string> } {
+  const startMoment = getStartOfWeek(store.currentTimezone);
+
+  const shifts: Array<{ shiftId: string; events: Event[] }> = false
```

Comment:
you don't need to check `false`

Reply:
ahhh, forgotten. It was for debugging, good catch, I missed it.